guides/v3.2.0/components/the-component-lifecycle.md

```diff
@@ -237,8 +237,8 @@ export default Component.extend({
 
   didReceiveAttrs() {
     this._super(...arguments);
-    this.set('items', this.get('items').map((item) => {
-      if (item.id === this.get('selectedItem.id')) {
+    this.set('items', this.items.map((item) => {
+      if (item.id === this.selectedItem.id) {
```

Comment:
Unrelated to this PR, but it seems like this `map()` is mutating the items. I think some kind of for…of or forEach could better express that we are changing the items in place. But no need to make the change in this PR unless you feel especially motivated to do so :-)

Reply:
I can look at it and propose a better example, no problem.